websites/M/MKPC/presence.ts

```diff
@@ -41,16 +39,26 @@ presence.on("UpdateData", async () => {
     user = document.querySelector("html > body > main > h1");
     presenceData.details = "Viewing the following category: " + user.innerHTML;
     presenceData.smallImageKey = "search";
+    presenceData.buttons = [{ label: "View category", url: document.location.href }];
   } else if (document.location.pathname == "/topic.php") {
     presenceData.startTimestamp = browsingStamp;
     user = document.querySelector("html > body > main > h1");
     presenceData.details = "Viewing: " + user.innerHTML;
     presenceData.smallImageKey = "search";
-  } else if (document.location.pathname == "/profil.php") {
+    presenceData.buttons = [{ label: "View topic", url: document.location.href }];
+  } else if (
+    document.location.pathname == "/ban-player.php" ||
+    document.location.pathname == "/admin.php" ||
+    document.location.pathname == "doublecomptes.php"
+  ) {
     presenceData.startTimestamp = browsingStamp;
-    user = document.querySelector("html > body > main > h1");
+    presenceData.details = "Viewing staff backend";
+  } else if (document.location.pathname == "/profil.php") { 
```

Comment:
shouldn't `profil` have an `e` at the end

Reply:
i know, its the location of the website thats spelled like this xd, even i was confused but the location is /profil.php